scripts/flow/environment.js

```diff
@@ -18,6 +18,8 @@ declare var __REACT_DEVTOOLS_GLOBAL_HOOK__: any; /*?{
   inject: ?((stuff: Object) => void)
 };*/
 
+declare var globalThis: Object;
+
```

Comment:
Isn't this like typing it as any when it is quite unsafe to do so? why not just suppress the flow warning where you attempt to reference it if you don't want to existence check it?